translations/tiled_de.ts

```diff
@@ -3899,7 +3900,7 @@ Bitte Format angeben.</translation>
     <message>
         <location line="+1"/>
         <source>Erase Terrain</source>
-        <translation type="unfinished"></translation>
+        <translation>Terrain löschen</translation>
```

Comment:
Ohh.... I thought the button would deleted an entire terrain entry... Probably because it is placed on the list and not in the toolbar. Why is it there while all other mapping tools are in the toolbar?